<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemsTableDockWidget</class>
 <widget class="QDockWidget" name="ItemsTableDockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>778</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Items Table</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Query:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_lineEdit_query" native="true"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_query_exec">
        <property name="text">
         <string>Execute</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_query_reset">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Limit:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_limit">
        <property name="toolTip">
         <string>Maximum number of rows per page to display (0 - no limit).</string>
        </property>
        <property name="maximum">
         <number>1000000</number>
        </property>
        <property name="singleStep">
         <number>20</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Page:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_page">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1000000</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="tableView_items"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
